### DISCUSSION TITLE
Add Apple Silicon Support for mpv Cask

### DISCUSSION THREAD

**Discussion by george-emerald** [Homebrew/discussions#5251] [Casks]
### Output of `brew config`

```shell
Irrelevant.
```

### Output of `brew doctor`

```shell
Irrelevant.
```

### Description of issue

The mpv formula has had Apple Silicon native support for a while now. But it's practically useless since you can only use it through CLI. Who wants to open the terminal and type out a command every single time he wants to watch a video? Can you please add native Apple Silicon support to the cask as well?

### Relevant casks

mpv

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew should help you do just that

**Comment by cho-m** [comment]
There is no build for Apple Silicon from a reliable source (i.e. listed on official page https://mpv.io/installation/). We use the 3rd party build from https://laboratory.stolendata.net/~djinn/mpv_osx/ which says:
> faq: "Could you provide a native build for M1/M2 Macs?" - i'll look into it once i get access to an M1/M2 Mac.

We can't add a non-existent URL to a Cask.

If `mpv` decides to upload official binaries (https://github.com/mpv-player/mpv/pull/13529), then those can be considered. But again nothing is available yet.

**Comment by george-emerald** [comment]
If there's no official build for Apple Silicon, where are you sourcing the Apple Silicon build in the mpv formula? 
https://formulae.brew.sh/formula/mpv

**Reply by cho-m** [reply]
Formulae are built from source. We do not provide the .app/GUI for reasons discussed in https://github.com/Homebrew/homebrew-cask/issues/86226

**Comment by porg** [comment]
After moving to an Apple Silicon Mac I strongly miss the GUI mpv.app there. Would love to be able to install it via Homebrew!
